src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![no_std]
-#![feature(allocator_api, const_fn, lang_items, alloc_error_handler)]
+#![feature(alloc, allocator_api, const_fn, lang_items, alloc_error_handler)]
```

Comment:
```suggestion
#![feature(allocator_api, const_fn, lang_items, alloc_error_handler)]
```

Reply:
Is alloc stable already?

Reply:
Yup -- if you check the CI it's failing because of that.